eclair-core/src/test/scala/fr/acinq/eclair/blockchain/bitcoind/BitcoinCoreClientSpec.scala

```diff
@@ -562,6 +537,153 @@ class BitcoinCoreClientSpec extends TestKitBaseClass with BitcoindService with A
     }, max = 10 seconds, interval = 1 second)
   }
 
+  test("unlock transaction inputs when double-spent by another wallet transaction") {
+    assume(!useEclairSigner)
+
+    val sender = TestProbe()
+    val priv = randomKey()
+    val miner = makeBitcoinCoreClient()
+    // We initialize our wallet with two inputs:
+    val wallet = new BitcoinCoreClient(createWallet("mempool_double_spend", sender))
+    wallet.getReceiveAddress().pipeTo(sender.ref)
+    val address = sender.expectMsgType[String]
+    Seq(200_000 sat, 200_000 sat).foreach(amount => {
+      miner.sendToAddress(address, amount, 1).pipeTo(sender.ref)
+      sender.expectMsgType[TxId]
+    })
```

Comment:
Nit (few occurences)
```suggestion
    Seq(200_000 sat, 200_000 sat).foreach { amount =>
      miner.sendToAddress(address, amount, 1).pipeTo(sender.ref)
      sender.expectMsgType[TxId]
    }
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/2817/commits/5ffbdb67e576ba54b3ee43be1217b9c1fba57a54